lib.sh

```diff
@@ -1128,8 +1143,7 @@ esac
 #
 # occ_command_no_check notification:generate -l "$2" "$admin" "$1"
 notify_admin_gui() {
-install_if_not jq
-if ! occ_command_no_check app:list --output=json | jq -e '.enabled | .notifications' > /dev/null
+if ! is_app_enabled notifications
```

Comment:
Maybe it's better to actually enable the app?

Reply:
I wouldn't say so because there are usecases where you would like to disable the notification app.